src/docfx/config/Config.cs

```diff
@@ -19,6 +19,11 @@ internal class Config : PreloadConfig
             "_themes/**",           // Default template location
         };
 
+        /// <summary>
+        /// Gets the default locale of this docset.
+        /// </summary>
+        public string DefaultLocale { get; private set; } = "en-us";
```

Comment:
Is any repository using this config?

Reply:
nop, this config is not used by docs right now.